src/Forms/FormConfig.php

```diff
@@ -193,6 +196,35 @@ public function getFormId()
         return 'form_'.$this->name;
     }
 
+    /**
+     * @param $value
+     * @return $this
+     */
+    public function setNavigationHeadingLevel($value)
```

Comment:
This limits you to H1-6.  Since this is a library, don't do that.

Reply:
Done!